code/game/machinery/doors/airlock.dm

```diff
@@ -123,12 +123,15 @@
 	var/abandoned = FALSE
 	var/cutAiWire = FALSE
 	var/autoname = FALSE
-	var/doorOpen = 'sound/machines/airlock.ogg'
-	var/doorClose = 'sound/machines/airlockclose.ogg'
```

Comment:
`sound/machines/airlock.ogg` and `sound/machines/airlockclose.ogg` should probably be replaced fully and deleted since everything's being moved into `sound/machines/doors`:
![open](https://user-images.githubusercontent.com/57483089/169140195-721926d4-7510-4d05-84a3-b59c77a33bb3.PNG)
![close](https://user-images.githubusercontent.com/57483089/169140196-3b001a4d-af5c-42cd-aee8-626a2d0775f4.PNG)